.github/workflows/pr_test_cpu.yml

```diff
@@ -17,21 +17,24 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: ['Ubuntu-latest', 'Windows-latest', 'MacOS-latest']
+        os: ['Ubuntu-latest', 'Windows-latest']
         pytorch-dtype: ['float32', 'float64']
-        exclude:
-          - os: 'Windows-latest'
-            pytorch-dtype: 'float64'
-          - os: 'MacOS-latest'
-            pytorch-dtype: 'float64'
 
     uses: kornia/workflows/.github/workflows/tests.yml@v1.6.0
     with:
       os: ${{ matrix.os }}
       python-version: '["3.8", "3.11"]'
-      pytorch-version: '["1.9.1", "2.2.0"]'
+      pytorch-version: '["1.9.1", "2.2.2"]'
```

Comment:
Update pytorch to 2.3.0 on [wokflows](https://github.com/kornia/workflows/tree/main) and here